hack/make-rules/test.sh

```diff
@@ -180,113 +173,72 @@ junitFilenamePrefix() {
   echo "${KUBE_JUNIT_REPORT_DIR}/junit_$(kube::util::sortable_date)"
 }
 
-produceJUnitXMLReport() {
-  local -r junit_filename_prefix=$1
-  if [[ -z "${junit_filename_prefix}" ]]; then
-    return
-  fi
-
-  local junit_xml_filename
-  junit_xml_filename="${junit_filename_prefix}.xml"
-
+installTools() {
```

Comment:
super nit, maybe a name more aligned to the other function conventions makes it more readable as `install::gotest::tools`

Reply:
I think local functions in a script don't need to follow such a convention. The other functions don't (`junitFilenamePrefix`, `runTests`).

Reply:
yeah, this is for "library" functions